### PR TITLE
#469 - fixed bug with year period

### LINKED ISSUE #469: When user A changes the year then user B also has changed year
Steps to reproduce:

1. As a user A change the test, to for example 2020
2. As a user B see that year is also changed

### LINKED ISSUE #470: Add button "Dzisiaj" to the calendar
After speaking with users, they'd like to have button "dzisiaj" - we can add it like this

image

### PR DESCRIPTION
**Pull request by kamilpiech97**
Should close #469.
Should close #470.

Preview of added button:
Screenshot from 2024-07-22 10-45-28

### CODE REVIEW

File: resources/js/Pages/Calendar.vue [hunk 1]
```diff
@@ -158,6 +160,17 @@ function linkVacationRequest(user) {
...
             <ChevronDoubleRightIcon class="w-5 h-5" />
           </span>
+          <InertiaLink
+            v-if="selectedMonth.value !== currentMonth || currentYear !== selectedYear"
+            :href="`/calendar/${props.currentMonth}/${props.currentYear}`"
+            as="button"
+            class="flex focus:relative justify-center items-center p-2 bg-white md:px-2 md:hover:bg-gray-50 ml-1"
+          >
+            <ArrowUturnLeftIcon class="w-5 h-5 text-blumilk-400 hover:text-blumilk-500" />
+            <span class="ml-1.5 text-sm text-blumilk-400 hover:text-blumilk-500">
```
Comment (EwelinaSkrzypacz) [inline resources/js/Pages/Calendar.vue:170]:
```suggestion
            <span class="ml-1.5 text-sm font-semibold text-blumilk-600 hover:text-blumilk-500">
```

File: resources/js/Pages/Calendar.vue [hunk 2]
```diff
@@ -158,6 +160,17 @@ function linkVacationRequest(user) {
           >
             <ChevronDoubleRightIcon class="w-5 h-5" />
           </span>
+          <InertiaLink
+            v-if="selectedMonth.value !== currentMonth || currentYear !== selectedYear"
+            :href="`/calendar/${props.currentMonth}/${props.currentYear}`"
+            as="button"
+            class="flex focus:relative justify-center items-center p-2 bg-white md:px-2 md:hover:bg-gray-50 ml-1"
+          >
+            <ArrowUturnLeftIcon class="w-5 h-5 text-blumilk-400 hover:text-blumilk-500" />
```
Comment (EwelinaSkrzypacz) [inline resources/js/Pages/Calendar.vue:169]:
```suggestion
            <ArrowUturnLeftIcon class="w-5 h-5 text-blumilk-600 hover:text-blumilk-500" />
```

**Review by EwelinaSkrzypacz — COMMENTED** [review]
When I click "Dzisiaj" I see toast - I don't know why, because I didn't change the yeat

image

**Comment by kamilpiech97** [comment]
> 1. When I click "Dzisiaj" I see toast - I don't know why, because I didn't change the year
> 
> image
> 
> 2. On calendar page I see warning in console
> 
> image
> 
> 3. We should change also notification when user edit details - from "Historia użytkownika utworzona" to "Szczegóły użytkownika utworzone" - the same thing with editing and deleting.
> 4. Maybe we should change icon for user details - as we change name from "Historia" to "Szczegóły" in my opinion to icon "information-circle" from heroicons

1,2,3,4 fixed

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by Baakoma — APPROVED** [review]

**Pull request merged**